test/database.test.ts

```diff
@@ -5,9 +5,10 @@ import { getAccounts } from "@tableland/local";
 import { getDefaultProvider } from "ethers";
 import { Database } from "../src/database.js";
 import { Statement } from "../src/statement.js";
+import { TEST_TIMEOUT_FACTOR } from "./setup";
 
 describe("database", function () {
-  this.timeout("10s");
+  this.timeout(TEST_TIMEOUT_FACTOR * 10000);
```

Comment:
This is the basic usage.  `TEST_TIMEOUT_FACTOR` defaults to `1` but can be set to anything greater than zero via an env var which lets us increase or reduce the timeout as needed.